### PR TITLE
Add in product changelog

### PR DESCRIPTION
**Pull request by mustard-mh**
## TODO
- [x] Use some endpoints (discussing) to query `latest publish date` and `markdown content`
- [x] Aligh to https://github.com/gitpod-io/openvscode-server/pull/381
- [x] Remove debug command

## Related Issue(s)

Fixes #

## How to test

- Follow steps in https://github.com/gitpod-io/gitpod-vscode-desktop/blob/master/docs/CONTRIBUTING.md
- Press `F5`
- Toggle command `Gitpod: Show Release Note`

## Release Notes

```release-note
NONE
```

## Incase you don't want to run it

|Screenshot| Light|
|-|-|
|image|image|

### CODE REVIEW

**Comment by jeanp413** [comment]
Could you add some CSS magic :lipstick: 
image

**Comment by mustard-mh** [comment]
> Could you add some CSS magic 💄

@jeanp413  It's a TODO 🙈, do you have any suggested markdown CSS?

**Comment by jeanp413** [comment]
lets's try first using the same css as vscode release notes an then we can iterate from there if necessary, check `src/vs/workbench/contrib/markdown/browser/markdownDocumentRenderer.ts` in vscode

**Comment by jeanp413** [comment]
Nice :ok_hand:, not sure about the image because it takes to much screen space but other than that it look really good

**Comment by jeanp413** [comment]
@loujaybee @akosyakov thoughs about the image, should we keep it or not?

File: package.json [hunk 1]
```diff
@@ -87,6 +89,16 @@
...
 				"title": "Export all logs"
+			},
+			{
+				"command": "gitpod.releaseNote",
+				"category": "Gitpod",
+				"title": "Show Release Note"
+			},
+			{
+				"command": "gitpod.cleanReleaseNoteCache",
+				"category": "Gitpod",
```
Comment (akosyakov) [inline package.json:100]: Debug actions should go under `Developer`

File: package.json [hunk 2]
```diff
@@ -87,6 +89,16 @@
 				"command": "gitpod.exportLogs",
 				"category": "Gitpod",
 				"title": "Export all logs"
+			},
+			{
+				"command": "gitpod.releaseNote",
```
Comment (akosyakov) [inline package.json:94]: `showReleaseNotes`

**Comment by akosyakov** [comment]
> @loujaybee @akosyakov thought about the image, should we keep it or not?

I think updates to the content should be done directly in Changelog. I would like to see list of highlights and call out to use latest versions in each changelog at the top. Images should go to specific highlights below. cc @Cahllagerfeld @mikenikles

File: package.json (hunk 1 above)
Comment (mustard-mh) [reply package.json:100]: I will remove it before merge

**Comment by akosyakov** [comment]
We should use https://code.visualstudio.com/api/advanced-topics/remote-extensions#sync-user-global-state-between-machines to ensure that a user don’t see already seen changelog across different machines (local and Gitpod workspaces) [Example](https://code.visualstudio.com/api/extension-capabilities/common-capabilities#setkeysforsync-example)

**Pull request merged**